webpack.config.js

```diff
@@ -155,8 +155,8 @@ module.exports = {
             options: {
               extract: true,
               spriteFilename: 'img/svg/icons.svg',
-              symbolId: (path) => {
-                const { name } = parse(path);
+              symbolId: path => {
```

Comment:
I don't this is good style as for multiple arguments you would still require brackets so it will be inconsistent

Reply:
I kind of like the concisness of `items.map(item => item.name)` and I've configured [arrow-parens](https://eslint.org/docs/rules/arrow-parens) to allow both styles as sometimes it may be needed to have the parens for additional clarity.

That said, I'm thinking about undoing that change if it's preferred to always have the parens.

Reply:
Undid the `arrow-parens` change.